packages/devextreme/js/common.d.ts

```diff
@@ -71,7 +71,7 @@ export type ButtonStyle = 'text' | 'outlined' | 'contained';
  * @public
  * @namespace DevExpress.common
  */
-export type ButtonType = 'danger' | 'default' | 'normal' | 'success';
+export type ButtonType = 'danger' | 'default' | 'normal' | 'success' | 'string';
```

Comment:
[incorrect solution]
You've just added one more possible value `"string"`, but we need this option to accept any string value besides predefined (how it's done for text above)